documentation/develop/temporary-installation-in-firefox.md

```diff
@@ -44,18 +46,19 @@ To install an extension temporarily:
 
 - open Firefox
 - enter "about:debugging" in the URL bar
+- click "This Firefox"
 - click "Load Temporary Add-on"
 - open the extension's directory and select any file inside the extension.
 
-The extension will be installed, and will stay installed until you restart Firefox.
+The extension installs and remains installed until you remove it or restart Firefox.
 
 <!-- Single Column Body Module -->
 
 <!-- Video Popup Thumbnail -->
 
 {% include modules/video-popup.html
     title="borderify WebExtension - part 1"
-    youtube_id="cer9EUKegG4"
+    youtube_id="J7el77F1ckg "
```

Comment:
Looks like the reason for the fail in circle-ci is the extra space at the end of this string.